spec/features/dashboard/collection_spec.rb

```diff
@@ -47,6 +47,7 @@
         expect(page).to have_link(admin_set_b.title.first)
         expect(page).not_to have_link(collection3.title.first)
         expect(page).not_to have_link(admin_set_a.title.first)
+        expect(page).to have_content("3 collections in the repository")
```

Comment:
@elrayle and I discussed this test and were brainstorming about ways to get a regression test in for this.

The current test checks that a count appears, but not that it's the correct "full" count rather than the paginated one. We have at least 5 options:

  - Create > 10 collections for the test;
    - This would probably be prohibitively slow.
  - Add a view spec for the index view with a mock response object and test the behavior there;
  - Stub the response object
    - Something like `allow_any_instance_of(@response.class).to receive(:total_count).and_return(1000)`;
  - Set `page_size` to 1 (how?) and leave this feature test otherwise the same;
  - Do nothing.

I'm open to any of these. @blancoj could you and @elrayle agree on a way forward?